test/suites/ACID.m

```diff
@@ -2742,3 +2743,21 @@
   xlim([-1, 2]);
 end
 % =========================================================================
+function [stat] = colorbars()
```

Comment:
The problem is that we might have subtle differences between the different cases. On the other hand it is just a pain. Maybe try once that the different hacks all work?